src/visitors/Tag.js

```diff
@@ -111,6 +131,14 @@ function build(node: Object, context: Context): JSXElement {
 
 const TagVisitor = {
   jsx(node: Object, context: Context) {
+
+    if (getInterpolationRefs(node.name)) {
+      const expr = context.getInterpolationByRef(node.name);
+      if (expr) {
+        return t.jSXExpressionContainer(expr);
+      }
+    }  
```

Comment:
This doesn't quite seem right to me.  If I'm correct, it means that something like:

```js
const x = pug`
  div
    ${Foo} blah blah blah
`;
```

Would render as:

```js
const x = <div>{Foo}</div>;
```

I think it should probably render as:

```js
const x = <div><Foo>blah blah blah</Foo></div>;
```

This would require us to verify that the interpolated expression is an Identifier with a capital first letter and throw an error if not, then pass a modified node down to `build(node, context)`